lib/services/insights/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "azure-insights",
+  "author": "Microsoft Corporation",
+  "contributors": [
+    "Kamath, Prarthana <pkamath@microsoft.com>",
+    "Vannelli, Leo <leovann@microsoft.com>",
+    "Shen, Andy <andyshen@microsoft.com>",
+    "Ruela, Vinicius <viruela@microsoft.com>",
+    "Moussa, George <georgenm@microsoft.com>",
+    "McKinlay, David <davmc@microsofto.com>",
+    "Kulkarni, Jagadish <jagadisk@microsoft.com>",
+    "Calderon-Meza, Guillermo <gucalder@microsoft.com>"
+  ],
+  "version": "0.0.1",
```

Comment:
Our current version for sdk for net is 0.7.7-preview. So I am going to use 0.7.7-pre here.